packages/admin-panel/src/pages/resources/EntityRelationsPage.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TreeResourcePage } from './TreeResourcePage';
+
+export const EntityRelationsPage = ({ getHeaderEl }) => (
+  <TreeResourcePage
+    title="Entity Relations"
+    endpoint="entityHierarchyTree"
```

Comment:
I'd like another opinion on this endpoint's name. I introduced it in a previous PR but the fact that it doesn't match the resource page ("Entity Relations") is kind of a smell.

Other names that could be considered:
1. `entityHierarchy`
2. `entityRelationsTree`

I am thinking that perhaps "entity hierarchy" is a tree of "entity relation" nodes.

I'm leaning towards `entityRelationsTree`. It may be nice to start a convention for naming tree-compatible endpoints using `{nodeType}Tree`.

Reply:
I like the page being called `EntityHierarchiesPage`? I guess the semantics in my mind are:
- entity hierarchy: the hierarchically structured collection of entities
- entity relations: given relations between entities

So if the page were a flat list of relations, I'd argue we call it `EntityRelationsPage`. But since it's a hierarchical view, `EntityHierarchiesPage` seems more apt

Reply:
I agree, I think we discussed naming the page `Entity Relations` early on but `Entity HIerarchies` seems more appropriate.